ddl/reorg.go

```diff
@@ -193,8 +193,13 @@ func getTableTotalCount(w *worker, tblInfo *model.TableInfo) int64 {
 	}
 	defer w.sessPool.put(ctx)
 
+	executor, ok := ctx.(sqlexec.RestrictedSQLExecutor)
+	// return PseudoRowCount to avoid panic for `ctx` which doesn't implement RestrictedSQLExecutor
```

Comment:
I think we can explain that the `ctx` used in the test did not implement this function like `We use `mock.Context` in the test, which does not implement RestrictedSQLExecutor.`

Reply:
done